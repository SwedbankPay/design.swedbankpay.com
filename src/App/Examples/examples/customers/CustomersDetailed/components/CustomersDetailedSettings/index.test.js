import React from "react";
import { shallow } from "enzyme";

import CustomersDetailedSettings from "./index";

describe("Examples: CustomerDetailedSettings", () => {
    it("is defined", () => {
        expect(CustomersDetailedSettings).toBeDefined();
    });

    it("renders", () => {
        const wrapper = shallow(<CustomersDetailedSettings />);

        expect(wrapper).toMatchSnapshot();
    });
});
