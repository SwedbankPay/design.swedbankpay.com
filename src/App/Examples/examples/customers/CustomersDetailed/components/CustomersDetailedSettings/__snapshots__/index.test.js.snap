// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Examples: CustomerDetailedSettings renders 1`] = `
<Fragment>
  <h3>
    Notifications
  </h3>
  <div
    className="row justify-content-around mt-3"
  >
    <div
      className="col-lg-auto m-2"
    >
      <div
        className="d-flex flex-column"
      >
        <Togglebox
          checked={true}
          id="customers-detailed-settings-toggle-1"
          label="Delivery status changes"
        />
        <Togglebox
          id="customers-detailed-settings-toggle-3"
          label="Critical delivery messages"
        />
      </div>
    </div>
    <div
      className="col-lg-auto m-2"
    >
      <Togglebox
        id="customers-detailed-settings-toggle-2"
        label="New inquiry messages"
      />
      <Togglebox
        checked={true}
        id="customers-detailed-settings-toggle-4"
        label="Inquiry status changes"
      />
    </div>
  </div>
  <h3>
    Newsletter and information
  </h3>
  <div
    className="row mt-4"
  >
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="0"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-0"
        label="Subscription #1"
      />
    </div>
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="1"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-1"
        label="Subscription #2"
      />
    </div>
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="2"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-2"
        label="Subscription #3"
      />
    </div>
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="3"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-3"
        label="Subscription #4"
      />
    </div>
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="4"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-4"
        label="Subscription #5"
      />
    </div>
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="5"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-5"
        label="Subscription #6"
      />
    </div>
    <div
      className="col-lg-3 col-md-4 col-sm-6"
      key="6"
    >
      <Checkbox
        checked={true}
        id="customers-detailed-settings-checkbox-6"
        label="Subscription #7"
      />
    </div>
  </div>
  <h3>
    Customer type
  </h3>
  <InputGroup
    prefixValue="Type"
    selectOptions={
      Array [
        "Long-term contract",
        "Short-term contract",
        "One time purchase",
      ]
    }
    type="select"
  />
  <button
    className="btn btn-primary"
  >
    Save
  </button>
</Fragment>
`;
